manifests/bootstrap_rpm.pp

```diff
@@ -56,14 +56,10 @@
     rhsm_port       => $rhsm_port,
     rhsm_prefix     => $rhsm_prefix,
   } ~>
-  certs_bootstrap_rpm { $candlepin_consumer_name:
-    dir              => $katello_www_pub_dir,
-    summary          => $candlepin_consumer_summary,
-    description      => $candlepin_consumer_description,
-    files            => ["${katello_rhsm_setup_script_location}:755=${katello_www_pub_dir}/${katello_rhsm_setup_script}"],
-    bootstrap_script => "/bin/bash ${katello_rhsm_setup_script_location}",
-    postun_script    => file('certs/postun.sh'),
```

Comment:
This is ugly and I didn't spot it initially, but it was still reading the file from the old module. After this is merged, please open a PR to delete it.